README.md

```diff
@@ -48,9 +60,9 @@ spec:
           servicePort: main-port
 ```
 
-The Application Load Balancer that will be created, will have both an HTTP listener and an HTTPS listener. The latter, will use the automatically selected certificate.
+The Application Load Balancer that will be created by the controller, will have both an HTTP listener and an HTTPS listener. The latter, will use the automatically selected certificate.
 
-Alternatively, the ARN of the desired certificate can be specified with an annotation like in the following example: 
+Alternatively, you can specify the Amazon Resource Name of the desired certificate can be specified with an annotation like in the following example:
```

Comment:
I would still add "(ARN)" after "Amazon Resource Name" as "ARN" is more readable for people familiar with AWS